account_invoice_auto_send_by_email/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Account Invoice Auto Send By Email",
+    "summary": "Invoice with the email transmit method are send automatically.",
+    "version": "14.0.1.0.0",
+    "category": "Accounting",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["account", "account_invoice_transmit_method", "queue_job"],
```

Comment:
I'm wondering, couldn't this module avoid depending on queue job and send the emails with the cron? Then could be extended by another module to generate the jobs?